<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelText_FPS_Rate">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>jf open 粉圓 1.1</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>FPS：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShow_FPS_Rate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>jf open 粉圓 1.1</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>411</width>
      <height>463</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutMain">
     <item>
      <layout class="QGridLayout" name="gridLayoutDisplay">
       <item row="0" column="2">
        <widget class="QLabel" name="labelShow_Angle_Horizon">
         <property name="font">
          <font>
           <family>jf open 粉圓 1.1</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>180</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelText_Angle_Horizon">
         <property name="font">
          <font>
           <family>jf open 粉圓 1.1</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>頭部水平角度：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="labelShow_Angle_Verticle">
         <property name="font">
          <font>
           <family>jf open 粉圓 1.1</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>90</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelText_Angle_Verticle">
         <property name="font">
          <font>
           <family>jf open 粉圓 1.1</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>頭部垂直角度：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="VerticleLayoutSetting">
       <item>
        <widget class="QLabel" name="labelText_Setting">
         <property name="font">
          <font>
           <family>jf open 粉圓 1.1</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>角度容許值：</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayoutSrcollBar">
         <item row="2" column="1">
          <widget class="QLabel" name="labelText_Angle_Threshold_Verticle">
           <property name="font">
            <font>
             <family>jf open 粉圓 1.1</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>垂直閥值：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacerScrollbar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>45</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="sliderBar_Threshold_Horizon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="sliderBar_Threshold_Verticle">
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
           <property name="sliderPosition">
            <number>20</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelText_Angle_Threshold_Horizon">
           <property name="font">
            <font>
             <family>jf open 粉圓 1.1</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>水平閥值：</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayoutDistract">
       <item row="0" column="0">
        <widget class="QLabel" name="labelText_Focus_Rate">
         <property name="font">
          <font>
           <family>jf open 粉圓 1.1</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>專注度：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelShow_Focus_Rate">
         <property name="font">
          <font>
           <family>jf open 粉圓 1.1</family>
           <pointsize>15</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>專注度：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="imageView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>701</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>jf open 粉圓 1.1</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>image</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelShow_Head_Direction">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>407</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>jf open 粉圓 1.1</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>目前頭部方向</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>設定</string>
    </property>
    <addaction name="actionReal_Time_Image"/>
    <addaction name="actionShow_Image"/>
    <addaction name="actionSwitch_To_RGB"/>
    <addaction name="actionSwitch_To_Gray"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReal_Time_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>影像</string>
   </property>
  </action>
  <action name="actionShow_Image">
   <property name="text">
    <string>顯示</string>
   </property>
  </action>
  <action name="actionSwitch_To_RGB">
   <property name="text">
    <string>輸出RGB</string>
   </property>
  </action>
  <action name="actionSwitch_To_Gray">
   <property name="text">
    <string>輸出灰階</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
